### ISSUE TITLE
Feature Request: Reply Feature

### ISSUE THREAD

**Issue by effakcuL** [quackduck/devzat#83]
While using devzat today we came up with the idea, that it would be nice to be able to reply to a comment of someone else!

It might be achievable through a small (like maybe 3 or 4 characters long) hash value shown in front of every message.
Typing 
```sh
ping 1234 response
```
 or something could work out nicely

**Comment by Arkaeriit** [comment]
In what kind of situation would this be better that using `@<username>` to respond to someone?

**Comment by effakcuL** [comment]
Currently we are 2 people active in the chat so referencing a person does not achieve anything. However, conversations tend to stray from one topics into multiple (at least for me) and sometimes responding to a specific comment makes it clear what you are talking about (it could also repeat what the other person said). We use that a lot in other group chats

**Comment by quackduck** [comment]
I use markdown quotes to do this. Adding a hash in front of a message could complicate the UI, right? I’ll try it out anyway: maybe make it an optional feature.

**Comment by Arkaeriit** [comment]
I agree with @quackduck on that one, it would be ugly to have all the time a number in front of the message if it is only used in a very small fraction of all exchanges.

**Comment by effakcuL** [comment]
Sadly I have to agree. From what I have seen so far I guess there is no way to have a personalized version of the chat, right? Everyone sees whatever is posted. Otherwise there could be settings to enable showing the has value and for example the time it was posted (and only people that enable it see this)

**Comment by quackduck** [comment]
No, that is possible!

**Comment by quackduck** [comment]
Another way to do this is to have some sort of trigger (tab-tab / left key / right key etc) and display a markdown formatted reply directly on the prompt. Should be fairly easy with the term lib's autocomplete callback.
